src/transforms/sort.jl

```diff
@@ -0,0 +1,74 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Sort(col₁, col₂, ..., colₙ; alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
+    Sort([col₁, col₂, ..., colₙ]; alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
+    Sort((col₁, col₂, ..., colₙ); alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
+
+Sort the table by the values of the selected columns `col₁`, `col₂`, ..., `colₙ`.
+The keyword arguments are the same as in the `sort` function.
+
+    Sort(regex; alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
+
+Sort the table by the values of the columns that match with `regex`.
+
+# Examples
+
+```julia
+Sort(:a)
+Sort(:a, :c, rev=true)
+Sort([1, 3, 5], by=row -> abs.(row))
+Sort(("a", "c", "e"))
+Sort(r"[ace]")
+```
+
+## Notes
+
+* The row passed to `by` kwarg is the selection row, not the table row.
```

Comment:
I don't understand this comment. The `by` keyword is usually a function that applies to the entries before the sorting, correct?